src/Core/Package.vala

```diff
@@ -607,12 +609,22 @@ public class AppCenterCore.Package : Object {
         uint score = 0;
         foreach (var query in queries) {
             var query_score = component.search_matches (query);
-
             if (query_score == 0) {
-                score = 0;
-                break;
+                var id_down = component.name.down ();
```

Comment:
Is this intended to be the same as `name_down`? Presumably you mean `component.id.down ()`?